metadataclient/tests/conftest.py

```diff
@@ -4,9 +4,9 @@
 import tzlocal
 import pytest
 import metadataclient.mds
-import metadataclient.commands
+# import metadataclient.commands
```

Comment:
I mean discard these changes.  They only went in when we were trying to run the tests locally